draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -337,35 +325,35 @@ HTTP features such as {{?HPACK=RFC7541}}.
 
 # Receiving RateLimit fields {#receiving-fields}
 
-A client MUST process the received `RateLimit` fields.
+A client MUST process the received RateLimit fields.
 
-A client MUST validate the values received in the `RateLimit` fields before using them
+A client MUST validate the values received in the RateLimit fields before using them
 and check if there are significant discrepancies
 with the expected ones.
-This includes a `RateLimit-Reset` moment too far in the future or a `service-limit` too high.
+This includes a RateLimit-Reset field moment too far in the future or a service-limit too high.
```

Comment:
How can a client know what is too far, or too high?  What are we trying to protect against here?

Reply:
Same as `Retry-After: 36500000` ~ 1year.